### ISSUE TITLE
MAUI WebView Apps do not allow iframe videos to run full screen

### ISSUE THREAD

**Issue by efonsecab** [dotnet/maui#8030]
### Description

In a MAUI Blazor app, when you try to make an iframe video to run full screen in android devices, 
nothing happens.

### Steps to Reproduce

1. Create a new MAUI Blazor app
2. In the Index page, add an iframe to a video. Example below
`<iframe width="684" height="385" src="https://www.youtube.com/embed/sGYIDKx6cY0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>`
3. Deploy the app to an android device.
4. Try to make the video to play full screen

Same thing happens with Azure Media Player

### Version with bug

6.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

Android 12

### Did you find any workaround?

No, I haven't found a workaround

### Relevant log output

Log when using Azure Media Player
[chromium] [INFO:CONSOLE(37)] "Uncaught (in promise) TypeError: fullscreen error", source: https://amp.azure.net/libs/amp/latest/azuremediaplayer.min.js (37)
[CCodecBufferChannel] [c2.qti.avc.decoder#386] DEBUG: elapsed: n=16 [in=0 pipeline=0 out=12]
[ViewRootImpl@c269f0c[MainActivity]] ViewPostIme pointer 0
[ViewRootImpl@c269f0c[MainActivity]] ViewPostIme pointer 1
[ViewRootImpl@c269f0c[MainActivity]] ViewPostIme pointer 0
[ViewRootImpl@c269f0c[MainActivity]] ViewPostIme pointer 1
[BufferPoolAccessor2.0] bufferpool2 0xb4000076260d9838 : 5(15667200 size) total buffers - 4(12533760 size) used buffers - 341/347 (recycle/alloc) - 9/345 (fetch/transfer)
[ViewRootImpl@c269f0c[MainActivity]] ViewPostIme pointer 0
[CCodecBufferChannel] [c2.qti.avc.decoder#386] DEBUG: elapsed: n=16 [in=0 pipeline=0 out=12]
[ViewRootImpl@c269f0c[MainActivity]] ViewPostIme pointer 1
[chromium] [INFO:CONSOLE(37)] "Uncaught (in promise) TypeError: Failed to execute 'exitFullscreen' on 'Document': Document not active", source: https://amp.azure.net/libs/amp/latest/azuremediaplayer.min.js (37)
Thread finished: <Thread Pool> #16
Thread finished: <Thread Pool> #18
Thread finished: <Thread Pool> #15

**Comment by Eilon** [comment]
@efonsecab thank you for the report. For this part:

> Try to make the video to play full screen

Do the usual controls for attempting to show full screen show up, but when you tap it, nothing happens? Or does the full screen button not even show up?

**Comment by jfversluis** [comment]
Maybe (partially) related to #4807 ?

**Comment by efonsecab** [comment]
> @efonsecab thank you for the report. For this part:
> 
> > Try to make the video to play full screen
> 
> Do the usual controls for attempting to show full screen show up, but when you tap it, nothing happens? Or does the full screen button not even show up?

Yes, the controls appear, it's just that the full screen action is not executing. I just confirmed the issue also happens when using Azure Media Player.
I updated the Relevant log output for this scenario.

**Comment by TanayParikh** [comment]
@efonsecab to help us isolate this issue, can you please confirm if the issue also reproduces in the base MAUI WebView (without Blazor)?

**Comment by efonsecab** [comment]
> @efonsecab to help us isolate this issue, can you please confirm if the issue also reproduces in the base MAUI WebView (without Blazor)?

Yes, same issue with a normal MAUI app with a WebView.

Seems related/duplicate of this:
https://github.com/dotnet/maui/issues/7720

**Comment by kristinx0211** [comment]
verified repro on android.

**Comment by Guandaru6967** [comment]
> Maybe (partially) related to #4807 ? @efonsecab did you also test on iOS or Windows? Does it work properly there?

Works fine on window. ```
<iframe allowfullscreen="allowfullscreen" ..../>
```

**Comment by vhugogarcia** [comment]
I'm facing this issue as well. I have on my Maui App 7.0.x a webview playing a video and it does not allow for full screen in Android. On iOS it works natively.

**Comment by vhugogarcia** [comment]
Hello friends, I found a solution and I wrote an article about how to implement it, hopefully you find it helpful :)

In this post, I will show you how to enable the feature allowing users to watch videos in full-screen mode on .NET MAUI WebViews for Android, using a simple custom handler and some native code. https://dev.to/vhugogarcia/net-maui-webviews-how-to-play-videos-in-full-screen-1j03

Thank you all!

**Comment by jfversluis** [comment]
The PR associated for this was merged so I'm assuming this is fixed in recent versions, if not, please let us know!
